util/log/clog.go

```diff
@@ -630,6 +630,7 @@ type loggingT struct {
 	// does not let us avoid the =true, and that shorthand is necessary for
 	// compatibility. TODO: does this matter enough to fix? Seems unlikely.
 	toStderr        bool          // The -logtostderr flag.
+	json            bool          // The -logjson flag.
 	alsoToStderr    bool          // The -alsologtostderr flag.
```

Comment:
I really like this.  It's at least a step in the right direction.
And from our discussion in the office, we're going to need to be able to specify more than one type of file logging, typically both proto and text.